src/libraries/Hooks.sol

```diff
@@ -161,12 +168,16 @@ library Hooks {
         IPoolManager.ModifyPositionParams memory params,
         bytes calldata hookData
     ) internal returns (bool shouldExecute) {
```

Comment:
and same name change on these functions? Just thinking so that they are all standardized

beforeModifyPosition -> beforeModifyLiquidity 
afterModifyPosition -> afterModifyLiquidity

Reply:
fixed 👍